src/main/java/edu/harvard/iq/dataverse/FilePage.java

```diff
@@ -1546,6 +1546,7 @@ public String getIngestMessage() {
     }
     
     //Determines whether this File uses a public store and therefore doesn't support embargoed or restricted files
+    @Deprecated(forRemoval = true, since = "2024-07-07")
```

Comment:
I'm not sure that we want to deprecate this method. Even if we deprecate the instance-wide `:PublicInstall` setting, note the code in the method defaults to `StorageIO.isPublicStore(...)`; i.e. individual stores can still be "public".

Reply:
Ah, good point. I got a bit overzealous here. Will revert.

Reply:
Fixed in a00d2700d7.